.github/workflows/deploy.yml

```diff
@@ -0,0 +1,147 @@
+name: Deploy
+on:
+  push:
+    branches:
+      - master
```

Comment:
I think we will only want releases on pushes to `v*` tags. No need to release for unversioned pushed to master.
```suggestion
```